src/custom/components/TransactionSettings/TransactionSetttingsMod.tsx

```diff
@@ -159,12 +159,17 @@ export default function SlippageTabs({
       <AutoColumn gap="sm">
         <RowFixed>
           <TYPE.black fontWeight={400} fontSize={14} color={theme.text2}>
-            Slippage tolerance
+            MEV protected slippage:
```

Comment:
Remove `:` to be inline with other fields

```suggestion
            MEV protected slippage
```